Misc/NEWS.d/next/Documentation/2019-05-08-17-54-02.bpo-36783.Z9b3mL.rst

```diff
@@ -0,0 +1 @@
+Add new references for C API Documentationadditions of Time_FromTimeAndFold and PyDateTime_FromDateAndTimeAndFold. Patch by Edison Abahurire.
```

Comment:
I would remove this blurb. @matrixise or someone can just add the "skip news" label, but this is just a fixup from the previous PR, which was merged a bit prematurely.

Reply:
Thanks @pganssle 
I have removed the blurb. Please review the change. 
@matrixise